opm/autodiff/BlackoilWellModel_impl.hpp

```diff
@@ -876,9 +876,13 @@ namespace Opm {
         } while (it < max_iter);
 
         if (converged) {
-            deferred_logger.debug("Well equation solution gets converged with " + std::to_string(it) + " iterations");
+            if (terminal_output_) {
```

Comment:
I guess you need to change the initialization of `terminal_output_` to achieve logging on all processes. Currently it is initialized by
```
terminal_output_ = false;
if (ebosSimulator.gridView().comm().rank() == 0)
  terminal_output_ = EWOMS_GET_PARAM(TypeTag, bool, EnableTerminalOutput);
```

Reply:
> I guess we have to have another variable apart from `terminal_output` to make this work as expected and/or refactor `gatherDeferredLogger`

@blattms thank you for the comment. The way parallel logging works, is that all processes get the messages of all processes. terminal_output can be true only on rank 0. Hence, the messages from all processes get logged only once. Please, feel free to correct me, if I am wrong.

Reply:
You are right. I oversaw that convergence is checked globally.